buildsitehandler.lua

```diff
@@ -358,6 +358,19 @@ function BuildSiteHandler:DontBuildOnMetalOrGeoSpots()
 	self:PlotAllDebug()
 end
 
+function BuildSiteHandler:BuilNearNano(builder,utype)
+	local p = nil
+	for id,nanoPos in pairs(self.ai.nanoList) do
+		p = ai.buildsitehandler:ClosestBuildSpot(builder, nanoPos, utype)
+		if p then 
+			EchoDebug('found Position for near nano turret at: ' .. nanoPos.x ..' ' ..nanoPos.z)
+			break 
+		end
+
+	end
+	return p
+end
+
```

Comment:
search for all the nano maybe can get computation problems... so maybe is better to save the position in order of building and start from the end of the list, in this manner we exclude the nanoturrets blocked in in a forest of other nano.
An  extreme  solution, if we noticed a great computing work, is save and use only the last one nano turret builded.
Try with this config please, or say me what do you think